modules/puppet_proxy/puppet_api.rb

```diff
@@ -9,6 +9,11 @@ class Proxy::Puppet::Api < ::Sinatra::Base
   inject_attr :environment_retriever_impl, :environment_retriever
   inject_attr :puppet_runner_impl, :puppet_runner
 
+  get "/" do
```

Comment:
I don't think returning version from a get to the module's root is a good idea. Using '/version' would be better and more consistent.